noxfile.py

```diff
@@ -140,6 +140,15 @@ def actionlint(session: nox.Session) -> None:
     )
 
 
+@nox.session
+def zizmor(session: nox.Session) -> None:
+    """
+    Ren zizmor, a Github Actions security checker
+    """
+    install(session, req="zizmor")
+    session.run("zizmor", "--persona=regular", ".github/workflows")
```

Comment:
Would it make sense to allow passing the persona as an arg?

It could be persona by default but perhaps there could be cases where I'd want to use this session to conduct an audit and set `--persona=auditor`. I think this flexibility would give the nox session approach more of an advantage over the workflow.